# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/wayland/moz.build
# Commit: 2a39a8e425b0
# Full Hash: 2a39a8e425b0824c670ba03452c437c3748c990b
# Author: Greg V <greg@unrelenting.technology>
# Date: 2020-01-06 21:54:03
# Regressor Bug: 1587360
# File Overlap Count: 1
# Description:
#   Bug 1587360 - Add support for idle-inhibit Wayland protocol r=stransky
#   
#   The idle-inhibit protocol allows idle wake locks on Wayland
#   without any of the supported D-Bus interfaces running.
#   
# ==============================================================================

diff -r 47592189b305 -r 2a39a8e425b0 widget/gtk/wayland/moz.build
--- a/widget/gtk/wayland/moz.build	Mon Jan 06 17:02:14 2020 +0000
+++ b/widget/gtk/wayland/moz.build	Mon Jan 06 16:11:19 2020 +0000
@@ -9,13 +9,15 @@
 
 SOURCES += [
     'gtk-primary-selection-protocol.c',
+    'idle-inhibit-unstable-v1-protocol.c',
     'linux-dmabuf-unstable-v1-protocol.c'
 ]
 
 EXPORTS.mozilla.widget += [
     'gbm.h',
     'gtk-primary-selection-client-protocol.h',
-    'linux-dmabuf-unstable-v1-client-protocol.h',
+    'idle-inhibit-unstable-v1-client-protocol.h',
+    'linux-dmabuf-unstable-v1-client-protocol.h'
 ]
 
 include('/ipc/chromium/chromium-config.mozbuild')
